templates/winserver2019/python/pre_python38.dockerfile.j2

```diff
@@ -0,0 +1,24 @@
+{% if HTTPS_PROXY is defined %}
+RUN powershell.exe -Command `
+	Invoke-WebRequest -URI https://github.com/python/cpython/archive/refs/tags/v3.8.12.zip -OutFile %TMP%\\python.zip -Proxy %HTTPS_PROXY% ; `
+	Expand-Archive -Path %TMP%\\python.zip -DestinationPath c:\\ -Force ; Remove-Item %TMP%\\python.zip -Force ; `
+	Invoke-WebRequest -URI https://www.python.org/ftp/python/3.8.10/python-3.8.10-embed-amd64.zip -OutFile %TMP%\\python-for-build.zip -Proxy %HTTPS_PROXY% ; `
```

Comment:
Mixing of Python 3.8.12 and 3.8.10. Is it done intentionally?

Reply:
Python >3.6 is required for build scripts: https://github.com/python/cpython/blob/main/PCbuild/readme.txt 
It will not be present in the built image.